choco/nunit-console.nuspec

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>nunit-console</id>
```

Comment:
According to chocos faq (https://github.com/chocolatey/choco/wiki/CreatePackages#naming-your-package) it says **Don’t use dots** so I followed that :)